build/docker/zap-baseline.py

```diff
@@ -68,7 +69,10 @@
 # Pscan rules that aren't really relevant, eg example alpha rules
```

Comment:
It took me a couple readings to figure out what you meant here, I think for clarity and others not as familiar with the tool an edit might be good:
> eg the examples rules in the alpha set